docs/UserGuide.adoc

```diff
@@ -341,7 +342,100 @@ Format: `exit`
+with the direction of the relationship specified.
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
+* `INDEXOFDESTINATION` refers to the index of the person to whom the relationship is directed.
+* `DIRECTION` referring to the direction of this relationship, can only be String input of `directed` or `undirected`.
+* If the `DIRECTION` is `directed`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` matters.
+* If the `DIRECTION` is `undirected`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` does not matter.
+* The index *must be a positive integer* `1, 2, 3, ...`
+****
+
+Examples:
+
+* `list` +
+`addRelationship 2 3 directed` +
+Adds a directed relationship starting from the 2nd person to the 3rd person in the list.
+* `list` +
+`addRelationship 2 3 undirected` +
+Adds an undirected relationship between the 2nd person and the 3rd person in the list.
+
+=== Add a named relationship
+
+Adds a relationship between two persons and gives this relationship a name. +
+Format: `addNamedRelationship INDEXOFTHESOURCE INDEXOFTHEDESTINATION DIRECTION NAME`
+
+****
+* Adds a relationship between `INDEXOFTHESOURCE` and `INDEXOFTHEDESTINATION` +
+with the direction of the relationship specified and name of the relationship given.
+* `INDEXOFSOURCE` refers to the index of the person from whom the relationship is +
+initiated as shown in the most recent listing.
+* `INDEXOFDESTINATION` refers to the index of the person to whom the relationship is directed.
+* `DIRECTION` referring to the direction of this relationship, can only be String input of `directed` or `undirected`.
+* `NAME` referring to the name of the relationship *can only be alphanumeric*.
+* If the `DIRECTION` is `directed`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` matters.
+* If the `DIRECTION` is `undirected`, the order of `INDEXOFSOURCE` and `INDEXOFDESTINATION` does not matter.
+* The index *must be a positive integer* `1, 2, 3, ...`
+****
+
+Examples:
+
+* `list` +
+`addRelationship 2 3 directed knows` +
+Adds a directed relationship starting from the 2nd person to the 3rd person in the list. +
+The relationship is named as `knows`.
+* `list` +
+`addRelationship 2 3 undirected cousins` +
+Adds an undirected relationship between the 2nd person and the 3rd person in the list. +
+The relationship is named as `cousins`.
+
+=== Add a confidence estimate
+
+Adds a confidence estimate for an attribute of a person. +
+Format: `addConfidenceEstimate INDEX TYPEOFATTRIBUTE ESTIMATE`
+
+****
+* `INDEX` refers to the index of the person whose attribute the user wants to add a confidence estimate to +
+as shown in the most recent listing.
+* `TYPEOFATTRIBUTE` refers to the `type` of the attribute that the confidence estimate is added to.
+* The `type` of the attribute refers to `Address`, `Email`, `Name`, `Phone`, `Tags`, `Relationships`.
```

Comment:
combine this line with the previous line for greater clarity, i.e. "`ATTRIBUTE_TYPE` refers to the `type` of attribute that the confidence estimate is added to (e.g. `Address`, `Email`, `Name`, `Phone`, `Tags`, `Relationships`)."